nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -0,0 +1,349 @@
+    Table table = catalog.loadTable(TABLE_IDENTIFIER);
+
+    GenericRecordBuilder recordBuilder =
+        new GenericRecordBuilder(AvroSchemaUtil.convert(schema, "test"));
+    List<GenericData.Record> records = new ArrayList<>();
+    records.add(recordBuilder.set("id", 1L).build());
+    records.add(recordBuilder.set("id", 2L).build());
+    records.add(recordBuilder.set("id", 3L).build());
+
+    String fileLocation = table.location().replace("file:", "") + "/data/file.avro";
+    try (FileAppender<GenericData.Record> writer = Avro.write(Files.localOutput(fileLocation))
+                                                       .schema(schema)
+                                                       .named("test")
+                                                       .build()) {
+      for (GenericData.Record rec : records) {
+        writer.add(rec);
+      }
+    }
+
+    DataFile file = DataFiles.builder(table.spec())
+                             .withRecordCount(3)
+                             .withPath(fileLocation)
+                             .withFileSizeInBytes(Files.localInput(fileLocation).getLength())
+                             .build();
+
+    table.newAppend().appendFile(file).commit();
+
+    String manifestListLocation =
+        table.currentSnapshot().manifestListLocation().replace("file:", "");
+
+    Assert.assertTrue(catalog.dropTable(TABLE_IDENTIFIER, false));
+    Assert.assertFalse(catalog.tableExists(TABLE_IDENTIFIER));
+
+    Assert.assertTrue(new File(fileLocation).exists());
+    Assert.assertTrue(new File(manifestListLocation).exists());
+  }
+
+
+  @SuppressWarnings("VariableDeclarationUsageDistance")
+  @Test
+  public void testDropTable() throws IOException {
+    Table table = catalog.loadTable(TABLE_IDENTIFIER);
+
+    GenericRecordBuilder recordBuilder =
+        new GenericRecordBuilder(AvroSchemaUtil.convert(schema, "test"));
+    List<GenericData.Record> records = new ArrayList<>();
+    records.add(recordBuilder.set("id", 1L).build());
+    records.add(recordBuilder.set("id", 2L).build());
+    records.add(recordBuilder.set("id", 3L).build());
+
+    String location1 = table.location().replace("file:", "") + "/data/file1.avro";
+    try (FileAppender<GenericData.Record> writer = Avro.write(Files.localOutput(location1))
+                                                       .schema(schema)
+                                                       .named("test")
+                                                       .build()) {
+      for (GenericData.Record rec : records) {
+        writer.add(rec);
+      }
+    }
+
+    String location2 = table.location().replace("file:", "") + "/data/file2.avro";
+    try (FileAppender<GenericData.Record> writer = Avro.write(Files.localOutput(location2))
+                                                       .schema(schema)
+                                                       .named("test")
+                                                       .build()) {
```

Comment:
We typically use a continuation indent of 2 indents / 4 spaces, rather than aligning with the previous method call.

Reply:
:+1: